README.md

```diff
@@ -107,6 +109,8 @@ type SourceFormat =
   | "webp";
 ```
 
+Only jpeg/png/webp/avif/heif/tiff will adjust quality.
```

Comment:
I think it makes more sense to move this to the description of the property itself to make clearer to which property it belongs.